Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -16,3 +16,5 @@ Icons have the following colors:
 * Blue #253AA7
 * Black #000000
 * Orange #F78616
+
+[Firestore Details](FirestoreDetails.md)
```

Comment:
This didn't link properly when I click on it, and I can't find the actual documentation page when I build it?

Reply:
When I click the link on my forked branch it works.